core/src/Kokkos_Complex.hpp

```diff
@@ -785,6 +785,14 @@ KOKKOS_INLINE_FUNCTION Kokkos::complex<RealType> log(
   return Kokkos::complex<RealType>(log(abs(x)), phi);
 }
 
+//! base 10 log of a complex number.
+template<typename RealType>
+KOKKOS_INLINE_FUNCTION Kokkos::complex<RealType> log10(
+    const complex<RealType>& x) {
+  using Kokkos::log;
```

Comment:
I wonder if this line is necessary...

Reply:
Not necessary no

Reply:
It shouldn't be necessary.

Reply:
Per your advice, I removed `using Kokkos::log`. Thanks !